### ISSUE TITLE
Move much of this content to 'data exploration' lesson

### ISSUE THREAD

**Issue by tracykteal** [Reproducible-Science-Curriculum/introduction-RR-Jupyter#34]
Rather than introducing the notebook at the beginning of the workshop and then talking about data organization, much of this content could move to 'data organization' to introduce the notebook when we first start to use it.

https://github.com/Reproducible-Science-Curriculum/data-exploration-RR-Jupyter/issues/29

**Comment by tracykteal** [comment]
Also related, update what is here to be more an introduction to the workshop and ideas rather than technical about the notebook.

**Comment by hlapp** [comment]
Yes, indeed. One of the things we've away with repeatedly from teaching the prototype is that people who are in the workshop are self-selected and don't need to be converted. Instead, they're here to see the how.

**Comment by burkesquires** [comment]
What if the organization lesson was conducted in a notebook, I think the Jupyter notebook could be particularly useful in providing a template for he project folder structure as a series of bash commands or python code that users can execute to create a hierarchy of folders.

**Comment by tracykteal** [comment]
Having the organization lesson in the notebook is an interesting idea. Let's work on that idea at the hackathon.
